fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -42,34 +60,28 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     The parent group for the Fidesctl CLI.
     """
 
-    local_commands = [evaluate, parse, view_config]
-    api_commands = [
-        annotate_dataset,
-        apply,
-        delete,
-        generate_dataset,
-        get,
-        init_db,
-        ls,
-        ping,
-        reset_db,
-        webserver,
-    ] + local_commands
-
     ctx.ensure_object(dict)
     config = get_config(config_path)
 
-    for command in local_commands:
+    for command in LOCAL_COMMANDS:
         cli.add_command(command)
 
     # If local_mode is enabled, don't add unsupported commands
     if not (local or config.cli.local_mode):
         config.cli.local_mode = False
-        for command in api_commands:
+        for command in API_COMMANDS:
             cli.add_command(command)
     else:
         config.cli.local_mode = True
     ctx.obj["CONFIG"] = config
 
     if not ctx.invoked_subcommand:
         click.echo(cli.get_help(ctx))
+
+
+# This is a special section used for auto-generating the CLI docs
+# This has to be done due to the dynamic way in which commands are added for the real CLI group
+cli_docs = cli
+
+for cli_command in API_COMMANDS:
+    cli_docs.add_command(cli_command)
```

Comment:
There's no way this can be combined with the loops on L66 and L72, just to prevent having to loop again?

Reply:
not that I can think of, because the autodocs plugin doesn't actually run any of the python commands (which is a good thing, i think) it just loads the module and inspects the attributes of the CLI command class

Reply:
hence why i noted the jankiness of it, I couldn't think of another way to do this, but this is a small price to pay for the flexibility we have in our CLI _and_ getting auto-documentation